src/modules/aws_ec2/mappers/instance.ts

```diff
@@ -105,7 +105,7 @@ export class InstanceMapper extends MapperBase<Instance> {
 
     // check if we can find the instance in database
     if (instance.InstanceId) {
-      const instanceObj = await this.module.instance.db.read(
+      const instanceObj = await this.db.read(
```

Comment:
Calling it directly like this will bypass the memoization cache, which can negatively impact performance, but will guarantee it's getting the latest value from the DB. Not sure if that's what you want to do here.

Reply:
Nevermind. We fixed this in the past and I forgot about it.